scripts/provision-wallet.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+WALLET_NAME=$1
+
+AGORIC_SDK=$(find ~ -type d -name "agoric-sdk" | head -n 1)
```

Comment:
is there an easier way? this searches everywhere

Reply:
This script is only run once by the oracle. When setting up such a setup, it is usually the case the node operators use a clean user/VM. It is not worth optimising this.